frontend/services/entities/software.ts

```diff
@@ -234,7 +234,37 @@ export default {
       true
     );
   },
+  editSoftwarePackage: (
+    softwareId: number,
+    data: IAddPackageFormData, // TODO: Update name in renaming
+    teamId?: number,
+    timeout?: number
+  ) => {
+    const { SOFTWARE_TITLE } = endpoints;
+
+    if (!data.software) {
+      throw new Error("Software package is required");
+    }
 
+    const formData = new FormData();
+    formData.append("software", data.software);
+    formData.append("self_service", data.selfService.toString());
+    data.installScript && formData.append("install_script", data.installScript);
+    data.preInstallQuery &&
+      formData.append("pre_install_query", data.preInstallQuery);
+    data.postInstallScript &&
+      formData.append("post_install_script", data.postInstallScript);
+    teamId && formData.append("team_id", teamId.toString());
```

Comment:
Per https://github.com/fleetdm/fleet/pull/21729/files#diff-7246bc304b15c8865ed8eaa205e9c244d0a0314e4bae60cf553dc06147c38b64R9065, this is supposed to be required, but I'm not sure how we specify "no team" if it's required? I guess on the backend I'll treat omitted as no team?

Reply:
@iansltx looking at other APIs, we're using team_id=0 for no team. I'll update my FE code later today, thanks for catching this!

TODO @RachelElysia